ffv1.md

```diff
@@ -304,6 +305,7 @@ AART:          Q_(1)[tl - t] +
 AART:          Q_(2)[t - tr] +
 AART:          Q_(3)[L - l]  +
 AART:          Q_(4)[T - t]
+Figure: Description of the coding of the Context. {#figureContext}
```

Comment:
Coding generally to me means storing in the bitstream, we do not really store the context. Rather the Context for each sample is a consequence of the previously stored samples.

Reply:
`coding` changed to `computing`.